src/main.cc

```diff
@@ -801,12 +801,38 @@ int main(int argc, char* argv[])
                    { "ui", { true, "set the type of user interface to use (ncurses, dummy, or json)" } },
                    { "l", { false, "list existing sessions" } },
                    { "clear", { false, "clear dead sessions" } },
-                   { "ro", { false, "readonly mode" } } }
+                   { "ro", { false, "readonly mode" } },
+                   { "help", { false, "display a help message and quit" } } }
     };
+
     try
     {
+        auto show_usage = [&](const ParameterDesc &param_desc)
```

Comment:
Its always going to be the local param_desc, so no need to pass it as a a parameter.